### PR TITLE
Wavelets

### PR DESCRIPTION
**Pull request by szympie**

### CODE REVIEW

File: FeatureExtraction/Wavelets/wavelets_features.py [hunk 1]
```diff
@@ -24,29 +24,36 @@ def get_wavelet_features(image: GreyscaleImage, mother_wavelet: str) -> dict:
...
-        max_value, avg_value, kurtosis, skewness = get_features_for_detail_coefficients(
-            detail_coefficients)
-        features = {mother_wavelet + "_" + name + "_max": max_value,
-                    mother_wavelet + "_" + name + "_avg": avg_value,
-                    mother_wavelet + "_" + name + "_kurtosis": kurtosis,
-                    mother_wavelet + "_" + name + "_skewness": skewness}
-        features_dict = {**features_dict, **features}
+        detail_features = get_features_for_detail_coefficients(
+            detail_coefficients, mother_wavelet, name)
+        features_dict = {**features_dict, **detail_features}
```
Comment (khrynczenko) [inline FeatureExtraction/Wavelets/wavelets_features.py]: I think this is no longer OrderedDict after such operation.

**Review by khrynczenko — CHANGES_REQUESTED** [review]
Tests fail.

File: FeatureExtraction/Wavelets/wavelets_features.py (hunk 1 above)
Comment (szympie) [reply FeatureExtraction/Wavelets/wavelets_features.py]: Fixed

Comment (mmyller) [reply FeatureExtraction/Wavelets/wavelets_features.py]: +1. Maybe a test?

Comment (szympie) [reply FeatureExtraction/Wavelets/wavelets_features.py]: Done

**Review by mmyller — APPROVED** [review]

**Review by khrynczenko — APPROVED** [review]

**Pull request merged**